gremlin/templates/daemonset.yaml

```diff
@@ -31,7 +31,6 @@ spec:
         {{- if .Values.gremlin.podLabels }}
         {{- toYaml .Values.gremlin.podLabels | nindent 8 }}
         {{- end }}
-      {{- if (or .Values.gremlin.podSecurity.seccomp.enabled (or .Values.gremlin.apparmor .Values.gremlin.installApparmorProfile) ) }}
```

Comment:
Without this condition, we may end up with `annotations:` (i.e. empty value), which is equivalent to no `annotations` key at all. These two outcomes are functionality equivalent in the context of creating/updating our daemonset here.